### ISSUE TITLE
Overlay does not reliably start

### ISSUE THREAD

**Issue by RespiceFinemII** [mumble-voip/mumble#2911]
Using version "mumble-1.3.0-1937-g289d0d4-snapshot" the overlay will sometimes load for Eve Online Launcher (and game), but not reliably. The "mumble-1.3.0-1937-g289d0d4-snapshot.winx64" version seems to work less often but has also worked at least once. I am launching Mumble first and the launcher second, I can repeat this a few times, sometimes, and still have the overlay working each time. Restarting the system will sometimes make it work again. I set up DebugView as instructed in another ticket and captured one instance of it working and another instance of it not working, both with the "mumble-1.3.0-1937-g289d0d4-snapshot" version.
Examples of DebugView log files for working and not working instances are attached, as well as dxdiag if needed.
[DxDiag.txt](https://github.com/mumble-voip/mumble/files/819841/DxDiag.txt)
[OverlayNotWorking.txt](https://github.com/mumble-voip/mumble/files/819839/OverlayNotWorking.txt)
[OverlayWorking.txt](https://github.com/mumble-voip/mumble/files/819840/OverlayWorking.txt)

**Comment by mkrautz** [comment]
Hi,

Can you perhaps re-test with debug output enabled for the overaly?

See https://wiki.mumble.info/wiki/Debugging#Enabling_More_Verbose_Logging_in_the_Overlay

I am not 100% sure where it needs to be for Eve. Basically, it should be next to the game's executable -- not the launcher, but the game itself.

**Comment by RespiceFinemII** [comment]
I just noticed that "mumble_ol_helper.exe" and "mumble_ol_helper_x64.exe" only occasionally launch with "mumble.exe" if I watch the details tab of Win 10 task manager as I launch mumble. They don't have a delayed start either.

Regarding the more verbose logging, I already had that file by the Eve executable. I tried putting it next to the launcher as well now. As far as I had seen, the overlay always worked in the launcher if it was going to work in-game, but now I found out a lot more I think.

In the attached "OverlayWorkingInLauncherNotInGame_v1" the overlay worked in the launcher, with "mumble_ol_helper.exe" running this time. It did not work in-game, but "mumble_ol_helper.exe" was still running. The game and launcher were both whitelisted in this instance, and  "mumble_ol_helper_x64.exe" did not launch. [OverlayWorkingInLauncherNotInGame_v1.txt](https://github.com/mumble-voip/mumble/files/819863/OverlayWorkingInLauncherNotInGame_v1.txt)

In the following instance the overlay worked in the launcher and in the game. The overlay was set to blacklist and both the launcher and game were not on the list. "mumble_ol_helper.exe" and  "mumble_ol_helper_x64.exe" both launched with "mumble.exe". [OverlayWorkingInLauncherAndInGame.txt](https://github.com/mumble-voip/mumble/files/819872/OverlayWorkingInLauncherAndInGame.txt)

In this instance neither of the overlay exe's launched with mumble and the overlay did not work in the launcher or in-game. The overlay was still set to blacklist. [OverlayNotWorkingInLauncherOrInGame.txt](https://github.com/mumble-voip/mumble/files/819881/OverlayNotWorkingInLauncherOrInGame.txt)

In this final instance, only "mumble_ol_helper_x64.exe" launched with mumble!  [OverlayNotWorkingInLauncherOrInGame_v2.txt](https://github.com/mumble-voip/mumble/files/819890/OverlayNotWorkingInLauncherOrInGame_v2.txt)

I apologize for some variation in the files; in some cases I only launched DebugView once I saw that mumble had correctly launched the overlay exe's.
